docs/en/latest/getting-started.md

```diff
@@ -21,247 +21,275 @@ title: Getting Started
+    "nodes": {
+      "httpbin.org:443": 1
+    } 
+  }
+}
+```
 
-- Upstream information:
+This Route means all inbound requests will be forwarded to the `httpbin.org:443` Upstream when they meets **ALL** these rules(matched requests):
 
-    Upstream is a virtual host abstraction that performs load balancing on a given set of service nodes according to configuration rules.
-    Thus a single upstream configuration can comprise of multiple servers which offers the same service. Each node will comprise of a key(address/ip : port) and a value(weight of the node).
-    The service can be load balanced through a round robin or consistent hashing (cHash) mechanism.
+- Request's HTTP method is `GET`;
+- Request has `Host` Header, and its value is `example.com`;
+- Request's path matches `/services/users/*`, `*` means all subpaths, like `/services/users/getAll?limit=10`.
 
-    When configuring a route you can either set the upstream information or use service abstraction to refer the upstream information.
+After this Route is created, we could use Apache APISIX's address to access our backend services(Upstream actually):
 
-- Plugins
+```bash
+$ curl -i -X GET "http://{APISIX_BASE_URL}/services/users/getAll?limit=10" -H "Host: example.com"
+```
 
-    Plugins allows you to extend the capabilities of APISIX and to implement arbitrary logic which can interface with the HTTP request/response lifecycle.
-    Therefore, if you want to authenticate the API then you can include the Key Auth plugin to enforce authentication for each request.
+This will be forward to `https://httpbin.org:443/getAll?limit=10` by Apache APISIX.
 
 ### Create an Upstream
 
-Execute the following command to create an upstream with the id of '50' in APISIX. Let's use the round-robin mechanism for load balancing.
+After reading the above section, we know we have to set the `Upstream` for `Route`. Just executing the following command to create one:
 
 ```bash
-curl "http://127.0.0.1:9080/apisix/admin/upstreams/50" -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+$ curl "http://127.0.0.1:9080/apisix/admin/upstreams/50" -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
 {
-    "type": "roundrobin",
-    "nodes": {
-        "httpbin.org:443": 1
-    }
+  "type": "roundrobin",
+  "nodes": {
+    "httpbin.org:443": 1
+  }
 }'
 ```
 
-### Add a Route to Access the Upstream
+We use `roundrobin` as our load balancer mechanism, and set `httpbin.org:443` as our Upstream target(backend server), and its ID is `50`. For more fields, please refer to [Admin API](./admin-api.md).
 
-By default APISIX proxies the request via the HTTP protocol. As our backend is hosted in a HTTPS environment, let's use the proxy-rewrite plugin to change the scheme to HTTPS.
+**NOTE:** `Create an Upstream` is not required actually, because we could use [Plugin](./architecture-design/plugin.md) to interceptor requests, but let's assume we need to set at least one `Upstream` in this guide.
```

Comment:
I don't understandard this NOTE.

Reply:
Avoid user misconception that it is required, because we could use `Plugin` to response directly 🤔